heidrun/indiana_dc.rb

```diff
@@ -0,0 +1,64 @@
+Krikri::Mapper.define(:indiana_dc,
+                      :parser => Krikri::OaiDcParser)
+  provider :class => DPLA::MAP::Agent do
+    uri 'http://dp.la/api/contributor/indiana'
+    label 'Indiana Memory'
+    #I'm guessing here as to the naming of the above things
+  end
+
+  dataProvider :class => DPLA::MAP::Agent do
+    providedLabel record.field('dc:contributor')
+    end
+
+  isShownAt :class => DPLA::MAP::WebResource do
+    uri record.field('dc:identifier')
+  end
+
+  preview :class => DPLA::MAP::WebResource do
+    uri record.field('dc:source')
+    
+
+  originalRecord :class => DPLA::MAP::WebResource do
+    uri record_uri
+  end
+
+  sourceResource :class => DPLA::MAP::SourceResource do
+  
+  collection :class => DPLA::MAP::Collection, :each => record.field('xmlns:header', 'xmlns:setSpec'), :as => :coll do
```

Comment:
1) The `:each` clause should be the following: `header.field('xmlns:set_spec')`
2) (minor) This line needs to be indented two more spaces, to line up with the `end` below it